src/Raven.Studio/typescript/commands/maintenance/disableAdminLogsMicrosoftCommand.ts

```diff
@@ -1,15 +0,0 @@
-import commandBase = require("commands/commandBase");
-import endpoints = require("endpoints");
-
-class disableAdminLogsMicrosoftCommand extends commandBase {
```

Comment:
Does it mean we'll not be able to configure it via the studio any longer?

Reply:
you can, it is now built-in, check SetLogsConfigurationOperation and the DTO there